node/src/transition_frontier/transition_frontier_reducer.rs

```diff
@@ -101,6 +103,18 @@ impl TransitionFrontierState {
                 }
                 state.sync = TransitionFrontierSyncState::Synced { time: meta.time() };
             }
+            TransitionFrontierAction::RpcRespondBestTip { peer_id } => {
```

Comment:
Same as the other action that was added to the transition frontier, I don't think this belongs here, but instead with the other p2p handler callbacks.